### PR TITLE
Add operators for Query section and create Templates section 

### PR DESCRIPTION
**Pull request by sinansonmez**
This PR
- enhance Query section by adding available operators
- creates a new section to further explain the usage of templates

Fixes https://github.com/silverbulletmd/silverbullet/issues/55

### CODE REVIEW

File: website/🔌 Query.md [hunk 1]
```diff
@@ -21,6 +21,19 @@ The query plug is a built-in plug implementing the `<!-- #query -->` mechanism.
...
+#### 2.1. Available query operators:
+* `=` equals
+* `!=` not equals
+* `<` less than
+* `<=` less than or equals
+* `>` greater than
+* `>=` greater than or equals
+* `=~` to match against a regular expression
+* `!=~` does not match this regular expression
+* `in` to specify multiple values in a `where` clause
```
Comment (sinansonmez) [inline website/🔌 Query.md:33]: In addition to the operators suggested by you in the issue, I saw `in` operator in the [query grammar](https://github.com/silverbulletmd/silverbullet/blob/main/packages/plugs/query/query.grammar). Please let me know if I explained the usage of `in` correctly.

File: website/🔌 Query.md [hunk 2]
```diff
@@ -66,89 +79,115 @@ country: Germany
...
 <!-- /query -->
  
 #### 4.2 Plugs’ data sources
-Certain plugs can also provide special data sources to query a certain data. Some examples are 
+Certain plugs can also provide special data sources to query a certain data. Some examples are:
 * [[🔌 Github]] provides `gh-pull` to query PRs for selected repo
 * [[🔌 Mattermost]] provides `mm-saved` to fetch (by default 15) saved posts in Mattermost
 
-For complete list of data sources, please check plugs’ pages.
+For a complete list of data sources, please check plugs’ pages.
```
Comment (Willyfrog) [inline website/🔌 Query.md:91]:
```suggestion
For a complete list of data sources, please check plugs’ own pages.
```
to differentiate from the [[plugs]] page

**Review by Willyfrog — APPROVED** [review]
Added a minor comment to try and clarify which page. But other than that it is cool to have this detail on the docs. Thanks!

File: website/🔌 Query.md (hunk 1 above)
Comment (zefhemel) [reply website/🔌 Query.md:33]: I'm not 100% if it's implemented, let's leave it out for now.

Comment (sinansonmez) [reply website/🔌 Query.md]: Done, thanks

**Comment by zefhemel** [comment]
Awesome once again. Thanks!

**Pull request merged**